src/app/components/Video/index.jsx

```diff
@@ -13,18 +21,20 @@ const Video = ({
   statsDestination,
   uiLocale,
 }) => (
-  <div>
-    <div>video pid: {pid}</div>
-    <div>kind: {kind}</div>
-    <div>title: {title}</div>
-    <div>holdingImageURL: {holdingImageUrl}</div>
-    <div>items: {JSON.stringify(items, null, 4)}</div>
-    <div>statsAppName: {statsAppName}</div>
-    <div>statsAppType: {statsAppType}</div>
-    <div>statsCountername: {statsCountername}</div>
-    <div>statsDestination: {statsDestination}</div>
-    <div>uiLocale: {uiLocale}</div>
-  </div>
+  <GridItemConstrainedLargeNoMargin>
```

Comment:
arbitrarily chose this grid item but I'm assuming we want some logic around this.